components/kyma-environment-broker/internal/environmentscleanup/service.go

```diff
@@ -50,7 +52,7 @@ func (s *Service) PerformCleanup() error {
 	var result *multierror.Error
 	shootsToDelete, err := s.getShootsToDelete(s.LabelSelector)
 	if err != nil {
-		log.Error(errors.Wrap(err, "while getting shoots to delete"))
+		s.logger.Error(errors.Wrap(err, "while getting shoots to delete"))
 		result = multierror.Append(result, err)
```

Comment:
as disscussed f2f we could refactor this - `getShootsToDelete` returns error only in case when Gardener service was unable to list Shoots - considering this, we can fail fast here and return this error.